fr/sidebars.json

```diff
@@ -835,6 +835,7 @@
                     "integrations/plugin-packs/procedures/hardware-storage-ibm-ts3200-snmp",
                     "integrations/plugin-packs/procedures/hardware-storage-ibm-ts3500-snmp",
                     "integrations/plugin-packs/procedures/hardware-storage-kaminario-restapi",
+                    "integrations/plugin-packs/procedures/hardware-storage-lenovo-iomega-snmp",
                     "integrations/plugin-packs/procedures/hardware-storage-lenovo-sseries-snmp",
                     "integrations/plugin-packs/procedures/hardware-storage-lenovo-iomega-snmp",
```

Comment:
Why do we have iomega twice here? Probably that the FR version already existed right?